jupyterhub/apihandlers/base.py

```diff
@@ -112,6 +114,7 @@ def user_model(self, user):
             'admin': user.admin,
             'groups': [ g.name for g in user.groups ],
             'server': user.url if user.running else None,
+            'server_state': None,
```

Comment:
After #1817, we shouldn't add more fields to the user model, only the server.state field.